### PR TITLE
data:snehap02

### PR DESCRIPTION
**Pull request by snehap02**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3781"></a>

### CODE REVIEW

File: data/snehap02.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name: "Sneha Purkayastha",
```
Comment (Panquesito7) [inline data/snehap02.json:2]: Missing closing string.
```suggestion
  "name": "Sneha Purkayastha",
```

File: data/snehap02.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Exploring the path of Web Development. Obsessed with tailwindcss, learning Javascript.",
+  "avatar: "https://github.com/snehap02.png",
```
Comment (Panquesito7) [inline data/snehap02.json:6]: Same here.
```suggestion
  "avatar": "https://github.com/snehap02.png",
```

**Review by Panquesito7 — DISMISSED** [review]

File: data/snehap02.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "bio": "Exploring the path of Web Development. Obsessed with tailwindcss, learning Javascript.",
+  "avatar": "https://github.com/snehap02.png",
+  "links": [
+  {
+    "name": "Connect with me on Twitter",
+    "url": "https://twitter.com/SnehaPurkayast8",
+    "icon": "FaTwitter"
+},
+{
+    "name": "Connect with me on LinkedIn"
```
Comment (kumarsonsoff3) [inline data/snehap02.json:14]: Missing comma  `,` there.
```suggestion
    "name": "Connect with me on LinkedIn",
```

File: data/snehap02.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "url": "https://www.linkedin.com/in/sneha-purkayastha-3b3aa1160/",
+    "icon": "FaLinkedin"
+},
+{
+    "name": "Follow me on Github"
```
Comment (kumarsonsoff3) [inline data/snehap02.json:19]: Another comma `,` is missing here. 
```suggestion
    "name": "Follow me on Github",
```

**Review by kumarsonsoff3 — DISMISSED** [review]
Please add the missing commas as suggested below. Then it should be good to go! 🛠

**Review by snehap02 — COMMENTED** [review]
I have changed the remaining error.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by snehap02** [comment]
Thank you :)

**Pull request merged**